templates/jobsystem.mustache

```diff
@@ -1,13 +1,13 @@
 <section id="oc-form">
   <div class="centered">
-
-    <h1>Opportunity Center Job System</h1>
-
+    <div class="button-wrapper center">
+      <a href="/" data-event-category="Assessment" data-event-action="Back to homepage" title="Return to home page" class="basic-button">Back to home page</a>
```

Comment:
Can you make the `data-event-category` = `Jobsystem` or something similar? Otherwise it'll probably look like it comes from another page.